modules/apps/portal-store/portal-store-jcr/src/main/resources/content/Language_ar.properties

```diff
@@ -4,7 +4,6 @@ jackrabbit-credentials-password=Jackrabbit Credentials Password (Automatic Copy)
 jackrabbit-credentials-username=Jackrabbit Credentials Username (Automatic Copy)
 jackrabbit-repository-home=Jackrabbit Repository Home (Automatic Copy)
 jackrabbit-repository-root=Jackrabbit Repository Root (Automatic Copy)
-jcr-store-configuration-name=تكوين مخزن زملاءهم (Automatic Translation)
```

Comment:
I'm missing Language.properties in this commit

Reply:
`Language.properties` has already with the right text at `master`, but the rest of the files hadn't gone through `buildLang`

Reply:
Got it